File: atcoder/abc075A_one_out_of_three.php
<?php
// unihernandez22
// https://atcoder.jp/contests/abc075/tasks/abc075_a
// implementation

$a = explode(" ",trim(fgets(STDIN)));
if ($a[0] == $a[1]) {
	echo $a[2];
} else if ($a[0] == $a[2]) {
	echo $a[1];
} else {
	echo $a[0];
}
?>

File: atcoder/abc075A_one_out_of_three.nim
# Vicfred
# https://atcoder.jp/contests/abc075/tasks/abc075_a
# implementation, sorting, greedy
import strutils, sequtils, algorithm

var line = stdin.readLine.split.map parseInt
line.sort

if line[0] == line[1]:
  echo line[2]
else:
  echo line[0]

